database/seeders/LoadTestSeeder.php

```
@@ -0,0 +1,313 @@
+                }
+            }
+        }
+
+        // Create some general tasks
+        $this->createGeneralTasks($matter, $filingEvent, $userLogins, $this->faker->numberBetween(0, 3));
+
+        return $matter;
+    }
+
+    private function createRenewalTasks(
+        Matter $matter,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$matter'. (undefined)

(UnusedFormalParameter)